modules/game/client/controllers/game.client.controller.js

```diff
@@ -18,18 +18,29 @@ angular.module('game').controller('GameController', ['$scope', '$location', '$do
     $scope.CanvasSettings = CanvasSettings;
     $scope.ChatSettings = ChatSettings;
     $scope.GameSettings = GameSettings;
-    $scope.TopicSettings = TopicSettings;
     $scope.Object = Object;
 
+    // Get a list of topics from the server
+    $http.get('/api/topics').success(function (response) {
+      console.log('Retrieved list of topics', response);
+      $scope.listOfTopics = response;
+    }).error(function (response) {
+      console.error('Failed to fetch topics', response);
+    });
+
     // Pregame settings for host to change
     $scope.chosenSettings = {
       numRounds : GameSettings.numRounds.default,
       roundTime : GameSettings.roundTime.default,
       timeAfterGuess : GameSettings.timeAfterGuess.default,
-      topicListName: TopicSettings.topicListName.default,
-      topicListDifficulty: TopicSettings.topicListDifficulty.default
+      topicListName: undefined,
```

Comment:
Uhhh this may warrant changing, not sure if leaving it as undefined is the best idea.

More difficult to establish a default topic list, now that it's not "guaranteed" to have certain topics available. Probably guarantee-able as long as the seed.js runs, but it seems bad to assume that...